### PR TITLE
fix(DB/Quest): It's a Fel Reaver, But with Heart

### PR DESCRIPTION
**Pull request by grimgravy**
## Changes Proposed:
-  Reference: [TC@C92909](https://github.com/TrinityCore/TrinityCore/issues/16607)
-  co-author: @Gargarensis

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/14974
- Closes https://github.com/chromiecraft/chromiecraft/issues/5005

## SOURCE: 
TrinityCore

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1685912891682130400.sql [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+INSERT INTO `smart_scripts` (`entryorguid`, `source_type`, `id`, `link`, `event_type`, `event_phase_mask`, `event_chance`, `event_flags`, `event_param1`, `event_param2`, `event_param3`, `event_param4`, `event_param5`, `action_type`, `action_param1`, `action_param2`, `action_param3`, `action_param4`, `action_param5`, `action_param6`, `target_type`, `target_param1`, `target_param2`, `target_param3`, `target_param4`, `target_x`, `target_y`, `target_z`, `target_o`, `comment`) VALUES
+(20287, 0, 0, 0, 54, 0, 100, 0, 0, 0, 0, 0, 0, 49, 0, 0, 0, 0, 0, 0, 21, 85, 0, 0, 0, 0, 0, 0, 0, 'Zaxxis Ambusher - On Just Summoned - Start Attacking');
+
+-- Send script event
+DELETE FROM `event_scripts` WHERE `id`=13052;
+INSERT INTO `event_scripts` (`id`, `delay`, `command`, `datalong`, `datalong2`, `dataint`, `x`, `y`, `z`, `o`) VALUES
+(13052, 2, 10, 20287, 180000, 0, 2547.08, 3982.24, 131.39, 2.01),
+(13052, 20, 10, 20287, 180000, 0, 2537.70, 3975.96, 130.40, 1.58),
+(13052, 35, 10, 20287, 180000, 0, 2506.46, 4008.93, 133.80, 6.19),
+(13052, 50, 10, 20287, 180000, 0, 2537.30, 4027.11, 135.50, 4.30);
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/rev_1685912891682130400.sql:15]: What's the source for these spawns?

Also I think we prefer using SAI over event_scripts. They are partly deprecated

Comment (grimgravy) [reply data/sql/updates/pending_db_world/rev_1685912891682130400.sql:15]: @Gultask, if there is any problem let me know

Comment (Gultask) [reply data/sql/updates/pending_db_world/rev_1685912891682130400.sql:15]: I just want to know the source for the spawn points 😛

Cmangos, TC, etc.

Comment (grimgravy) [reply data/sql/updates/pending_db_world/rev_1685912891682130400.sql:15]: It's in discretion

**Comment by Gultask** [comment]
Oh sorry I missed that. Ok, in that case you need to add Gargarensis as a co-author for this PR

**Comment by FrancescoBorzi** [comment]
@Gultask what's the status of this?

**Comment by GastonCK** [comment]
Tested, working good.
Ambushers spawn every few seconds
WoWScrnShot_070123_115437

All remaining ambushers despawn when you kill the fel reaver, loot it and complete quest.
WoWScrnShot_070123_115717

**Review by Gultask — APPROVED** [review]

**Pull request merged**